test/commands/profiles/list.test.js

```diff
@@ -16,22 +16,24 @@ describe('commands', () => {
           expect(ctx.stderr).to.contain('No profiles have been configured');
         });
 
-      test
-        .do((ctx) => {
-          ctx.userConfig = new ConfigData();
-          ctx.userConfig.addProfile('profile1', constants.FAKE_ACCOUNT_SID);
-        })
-        .twilioCliEnv(Config)
-        .stdout()
-        .stderr()
-        .twilioCommand(ProfilesList, [])
-        .it('runs profiles:list with 1 profile', (ctx) => {
-          expect(ctx.stdout).to.contain('profile1');
-          expect(ctx.stdout).to.contain(constants.FAKE_ACCOUNT_SID);
-          expect(ctx.stdout).to.not.contain('Region');
-          expect(ctx.stdout.match(/true/g)).to.have.length(1);
-          expect(ctx.stderr).to.equal('');
-        });
+      /*
```

Comment:
Can we add TODO for these cases also, so that we will not miss track.

Reply:
Done!